colon-kw/lang/reader.rkt

```diff
@@ -32,5 +33,9 @@
      (lambda (key defval)
        (define (fallback) (if proc (proc key defval) defval))
        (case key
+         [(color-lexer)
+          (make-colon-kw-lexer
+           (or (fallback)
```

Comment:
Re: This is independent of what the base lexer does...
If this `(fallback)` call returns an actual lexer, it won't be the racket-lexer that it delegates to. If that new base lexer defines it's own `:` syntax, this will do the wrong thing.